install.sh

```diff
@@ -994,11 +994,24 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
-- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
-    (echo; echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"') >> ${shell_profile}
-    eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"
+# show different instructions on adding Homebrew to path based on if the user has already done so
+if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}"
+then
+  if ! type brew &>/dev/null
```

Comment:
```suggestion
  if ! type -P brew &>/dev/null
```

add `-P` option for force `PATH` lookup to avoid looking for alias and functions.